go.mod

```diff
@@ -41,7 +41,7 @@ replace golang.zx2c4.com/wireguard v0.0.20200121 => golang.zx2c4.com/wireguard v
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Jeffail/gabs v1.4.0
-	github.com/aws/aws-sdk-go v1.30.15
+	github.com/aws/aws-sdk-go v1.44.9
```

Comment:
This was not required but Gary Taylor said to do it.

Reply:
If this causes a ripple effect of forcing an upgrade in telegraf and is not explicitly required for the change in max dimensions, I wouldn't do this upgrade now.